tests/conftest.py

```diff
@@ -19,6 +22,40 @@ def _test_environment(monkeypatch):
     monkeypatch.setenv("AWS_SESSION_TOKEN", "testing")
 
 
+@pytest.fixture
+def source_config():
+    with smart_open.open("tests/fixtures/source_config.json", "r") as file:
+        return yaml.safe_load(file)
+
+
+@pytest.fixture
+def source_metadata_csv():
+    with open("tests/fixtures/source_metadata.csv") as file:
+        reader = csv.DictReader(file)
+        yield reader
+
+
+@pytest.fixture
+def source_metadata_csv_with_bitstreams():
```

Comment:
Could we just use this fixture and remove `source_metadata_csv`?

Reply:
I prefer to have two separate fixtures for this; `source_metadata_csv_with_bitstreams` represents the CSV file created by `helpers.update_metadata_csv()`, while `source_metadata_csv` represents the original metadata CSV file passed to the `reconcile` command.